web/samples_index/lib/src/samples.yaml

```
@@ -666,6 +666,7 @@ samples:
     platforms:
       - Windows, macOS, Linux, web, Android, iOS
     links:
-      - https://codelabs.developers.google.com/codelabs/flutter-next-gen-uis
+      - href: https://codelabs.developers.google.com/codelabs/flutter-next-gen-uis
+        text: Building next generation UIs in Flutter
```

Since this is already the title of the sample, maybe make it clear this a link to the codelab?

```suggestion
        text: Codelab
```